src/Tizen.NUI/src/internal/Interop/Interop.ImageLoading.cs

```diff
@@ -35,6 +35,21 @@ internal static partial class ImageLoading
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_LoadImageFromFile__SWIG_4")]
             public static extern global::System.IntPtr LoadImageFromFile(string jarg1);
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_LoadImageFromMemory__SWIG_0")]
```

Comment:
please be sure that the dali patches that you have mentioned in description should be merged prior to this patch is merged.
otherwise when the newly added property is used, there could be a malfunction.

Reply:
Dali patches must be merges before this patch.
I'll update this PR again after that patches append with your reviews.